tvshows/src/components/features/shows/ShowReviewSection/ShowReviewSection.tsx

```diff
@@ -1,33 +1,36 @@
 "use client";
-import { Fragment, useEffect, useState } from "react";
+import { useEffect, useState } from "react";
 import { IReview, IReviewList } from "@/typings/review.types";
 import { ReviewForm } from "../ReviewForm/ReviewForm";
 import { ReviewList } from "../../review/ReviewList/ReviewList";
 import { ShowDetailsCard } from "@/components/shared/ShowDetailsCard/ShowDetailsCard";
-import { IShow } from "@/typings/show.types";
+import { Container } from "@chakra-ui/react";
+import useSWR from "swr";
+import { getShowList } from "@/fetchers/show";
+import { useParams } from "next/navigation";
 
 const mockReviewList: IReviewList = {
   title: "Reviews",
   reviews: [],
 };
 
-const myShow: IShow = {
-  title: "The Simpsons",
-  description:
-    "The Simpsons uses the standard setup of a situational comedy, or sitcom, as its premise. The series centers on a family and their life in a typical American town, serving as a satirical parody of a middle class American lifestyle.",
-  image_url: "/images/simpsons.jpg",
-  image_alt: "Photo of The Simpsons",
-  averageRating: undefined,
-};
-
 export const ShowReviewSection = () => {
+    const params = useParams();
+
+  const {
+    data: showListResponse,
+    error,
+    isLoading,
+  } = useSWR(`/shows/${params?.id}`, () => getShowList(params?.id as string));
```

Comment:
getShowList is typed like it returns a list of shows, doesn't it return a single show?